src/components/molecules/LogOutput/LogOutput.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useEffect, useState} from 'react';
+
+import {LogAction} from '@models/log';
+
+import {useAppDispatch} from '@redux/hooks';
+import {setLogOutput} from '@redux/reducers/configSlice';
+
+import {StyledLogOutputContainer, StyledLogTextContainer, StyledPreLogText} from './LogOutput.styled';
+import LogOutputHeader from './LogOutputHeader';
+
+export type LogActionProps = {
+  logOutput: string;
+};
+
+type LogOutputProps = {
+  logOutput?: string;
+  executionId?: string;
+  actions?: Array<LogAction>;
+  isFullScreen?: boolean;
+};
+
+const LogOutput: React.FC<LogOutputProps> = props => {
+  const {logOutput = '', executionId, actions = ['copy', 'fullscreen'], isFullScreen = false} = props;
+
+  const dispatch = useAppDispatch();
+
+  const [logs, setLogs] = useState(logOutput);
+
+  useEffect(() => {
+    if (!logOutput && executionId) {
```

Comment:
Maybe here use something like `if( logOutput || !exectutionId) return;` and then continue with all the logic below